config/code_schools.yml

```diff
@@ -502,3 +502,15 @@
   va_accepted: true
   full_time: true
   hardware_included: false
+  
+  - name: Code Immersives
```

Comment:
There was a problem loading the config and I think it might have to do with this line being indented. Can you update this pull request to remove the indentation?